yazi-cli/src/main.rs

```diff
@@ -71,10 +71,16 @@ async fn run() -> anyhow::Result<()> {
 				package::Package::load().await?.print()?;
 			} else if cmd.upgrade {
 				package::Package::load().await?.install(true).await?;
-			} else if let Some(repo) = cmd.add {
-				package::Package::load().await?.add(&repo).await?;
-			} else if let Some(repo) = cmd.delete {
-				package::Package::load().await?.delete(&repo).await?;
+			} else if let Some(repos) = cmd.add {
+				let mut package = package::Package::load().await?;
+				for repo in repos {
+					package.add(&repo).await?;
```

Comment:
Please add a new method `add_many` in `yazi-cli/src/package/add.rs` and call it instead